web/src/pages/Main.tsx

```diff
@@ -34,30 +34,43 @@ const DividerWithLargeSpacings = styled(Divider)`
 
 export const MainPage = (props: { mercuryApi: MercuryAPI }): JSX.Element => {
   const { mercuryApi } = props
-  const [components, setComponents] = useState<ComponentResponse>()
+  const [componentProperties, setComponentProperties] =
+    useState<TComponentProperties>()
   const [isLoading, setIsLoading] = useState<boolean>(true)
   const [feedFlow, setFeedFlow] = useState<TFeedFlow>({
     unit: 'Sm3/d',
     value: 1000,
   })
-  const [componentComposition, setComponentComposition] =
-    useState<TComponentComposition>()
-  const [result, setResult] = useState<MultiflashResponse>()
+  const [usedComponentRatios, setUsedComponentRatios] = useState<
+    TComponentRatios | undefined
+  >()
+  const [result, setResult] = useState<MultiflashResponse>({
```

Comment:
IMO its better to leave this to undefined instead of setting a mock result.
Typescript will then help us to not do anything with the result before it has some actual value

Reply:
Right, yes, I agree completely. It seems to have reappeared when solving a merge conflict.